src/syntax/LiteralToken.js

```diff
@@ -123,6 +119,10 @@ class StringParser {
         return ch;
     }
   }
+
+  [Symbol.iterator]() {
```

Comment:
I guess you mean: In V8 we get better performance is the properties with computed names are added after all of the static names.

I think this is worth a comment, at least until its common knowledge or we have a lot of such cases.